tempesta_fw/http_parser.c

```diff
@@ -103,6 +103,19 @@ memchreol(const unsigned char *s, size_t n)
 #define IS_WS(c) (c == ' ' || c == '\t')
 
 /**
+ * When parsing it's necessary to not forget to update @data and @len
+ * properly. So, presence of __data_{} macroses will be helpful to reduce amount
```

Comment:
`presence of __data_{} macroses will be helpful to reduce amount` -> `__data_{} macros help to reduce the amount`.
Usually I try my best not to fix comments. Sorry about that.